### ISSUE TITLE
Provide downloads for polygons of supported grids

### ISSUE THREAD

**Issue by MattBlissett** [gbif/occurrence-cube#25]
Several people have asked for polygons of the grids for use in analysis.

I think Geopackage and/or Shapefiles would be sufficient.  There are EEA ones online somewhere.

**Comment by damianooldoni** [comment]
Indeed, they can be downloaded from the Data Hub of the EU: https://www.eea.europa.eu/en/datahub/datahubitem-view/3c362237-daa4-45e2-8c16-aaadfb1a003b

**Comment by MattBlissett** [comment]
I've improved this with [instructions](https://github.com/gbif/occurrence-cube/tree/master/grid-generator) for generating custom grids, as well as more global grid downloads provided by GBIF.
